uikit.json

```diff
@@ -1 +1 @@
_modules/@gov.au/link-list","settings":{"version":"1.0.0","plugins":["@gov.au/pancake-sass","@gov.au/pancake-react","@gov.au/pancake-json"],"sass":{"path":"lib/sass/_module.scss","sass-versioning":true},"react":{"path":"lib/js/react.js"}}},"@gov.au/page-alerts":{"name":"@gov.au/page-alerts","version":"1.0.0","dependencies":{"@gov.au/core":">=1.0.0","@gov.au/body":">=1.0.0"},"path":"/Users/alexpage/Sites/dta/furnace/node_modules/@gov.au/page-alerts","settings":{"version":"1.0.0","plugins":["@gov.au/pancake-sass","@gov.au/pancake-react","@gov.au/pancake-json"],"sass":{"path":"lib/sass/_module.scss","sass-versioning":true},"react":{"path":"lib/js/react.js"}}},"@gov.au/progress-indicator":{"name":"@gov.au/progress-indicator","version":"2.0.0","dependencies":{"@gov.au/core":">=1.0.0"},"path":"/Users/alexpage/Sites/dta/furnace/node_modules/@gov.au/progress-indicator","settings":{"version":"1.0.0","plugins":["@gov.au/pancake-sass","@gov.au/pancake-react","@gov.au/pancake-json"],"sass":{"path":"lib/sass/_module.scss","sass-versioning":true},"react":{"path":"lib/js/react.js"}}},"@gov.au/responsive-media":{"name":"@gov.au/responsive-media","version":"1.0.0","dependencies":{"@gov.au/core":">=1.0.0"},"path":"/Users/alexpage/Sites/dta/furnace/node_modules/@gov.au/responsive-media","settings":{"version":"1.0.0","plugins":["@gov.au/pancake-sass","@gov.au/pancake-json"],"sass":{"path":"lib/sass/_module.scss","sass-versioning":true}}},"@gov.au/select":{"name":"@gov.au/select","version":"1.0.0","dependencies":{"@gov.au/core":">=1.0.0"},"path":"/Users/alexpage/Sites/dta/furnace/node_modules/@gov.au/select","settings":{"version":"1.0.0","plugins":["@gov.au/pancake-sass","@gov.au/pancake-react","@gov.au/pancake-json"],"sass":{"path":"lib/sass/_module.scss","sass-versioning":true},"react":{"path":"lib/js/react.js"}}},"@gov.au/skip-link":{"name":"@gov.au/skip-link","version":"1.0.0","dependencies":{"@gov.au/core":">=1.0.0"},"path":"/Users/alexpage/Sites/dta/furnace/node_modules/@gov.au/skip-link","settings":{"version":"1.0.0","plugins":["@gov.au/pancake-sass","@gov.au/pancake-react","@gov.au/pancake-json"],"sass":{"path":"lib/sass/_module.scss","sass-versioning":true},"react":{"path":"lib/js/react.js"}}},"@gov.au/tags":{"name":"@gov.au/tags","version":"1.0.0","dependencies":{"@gov.au/core":">=1.0.0"},"path":"/Users/alexpage/Sites/dta/furnace/node_modules/@gov.au/tags","settings":{"version":"1.0.0","plugins":["@gov.au/pancake-sass","@gov.au/pancake-react","@gov.au/pancake-json"],"sass":{"path":"lib/sass/_module.scss","sass-versioning":true},"react":{"path":"lib/js/react.js"}}},"@gov.au/text-inputs":{"name":"@gov.au/text-inputs","version":"1.0.0","dependencies":{"@gov.au/core":">=1.0.0"},"path":"/Users/alexpage/Sites/dta/furnace/node_modules/@gov.au/text-inputs","settings":{"version":"1.0.0","plugins":["@gov.au/pancake-sass","@gov.au/pancake-react","@gov.au/pancake-json"],"sass":{"path":"lib/sass/_module.scss","sass-versioning":true},"react":{"path":"lib/js/react.js"}}}}
```

Comment:
This file is generated by pancake-json right? Should be ignored from git

Reply:
Definitely should be ignored